src/reducers/entities/channels.js

```diff
@@ -492,6 +501,21 @@ function totalCount(state = 0, action) {
     }
 }
 
+export function manuallyUnread(state = {}, action) {
+    switch (action.type) {
+    case ChannelTypes.REMOVE_MANUALLY_UNREAD: {
+        if (state[action.data.channelId]) {
+            return {...state, [action.data.channelId]: false};
```

Comment:
even though I think this is fine, maybe consider removing the channel from the state

Reply:
Sure!